packages/cli-kit/src/testing/ui.ts

```diff
@@ -1,22 +1,18 @@
-import * as path from '../path.js'
-import {ExecaChildProcess, execaNode} from 'execa'
+export function waitForInputsToBeReady() {
```

Comment:
😬 Is there are way to subscribe to the "readiness"? I'm afraid this will result in flakiness (e.g. when running the tests in environments where the system needs more time for the input to be ready).